src/System.Net.Security/src/project.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "System.Diagnostics.Contracts": "4.0.0-*",
+    "System.Threading": "4.0.0-*",
+    "System.Net.Primitives": "4.0.10-beta-*"
```

Comment:
cc: @ericstj 

I'm not sure we should still be basing on "-beta-" in the version fields anymore.